cadquery/occ_impl/shapes.py

```diff
@@ -114,7 +114,7 @@
 
 TOLERANCE = 1e-6
 DEG2RAD = 2 * pi / 360.
-HASH_CODE_MAX = int(1e+6)  # required by OCC.Core.HashCode
+HASH_CODE_MAX = 2147483647  # MAX_INT, required by OCC.Core.HashCode
```

Comment:
Could you maybe add: "max 32bit signed int" in the comment?

Reply:
Yes, makes sense.